docs/user-guide/federation/index.md

```diff
@@ -46,3 +46,14 @@ The following guides explain some of the resources in detail:
 
 [API reference docs](/federation/docs/api-reference/readme/) lists all the
 resources supported by federation apiserver.
+
+## Cascading deletion
+
+Kubernetes v1.5 includes alpha support for cascading deletion of federated
+ingresses, namespaces, replica sets, secrets, deployments and daemon sets
+using `orphanDepedents` delete options.
```

Comment:
I think this is now way too long for the attention span of most people. But I will let docs reviewers to decide on that.